pkg/homedir/homedir.go

```diff
@@ -15,6 +15,14 @@ func Get() string {
 	return os.Getenv("HOME")
 }
 
+func Set(val string) {
```

Comment:
this is duplication. exact same if block exist for `Get()`, consider extracting into `EnvironmentKey()` method.